api/config/config_stable.json

```diff
@@ -0,0 +1,22 @@
+{
+  "firecloud": {
+    "debugEndpoints": false,
+    "billingAccountId": "014D91-FCB792-33D2C0",
+    "billingProjectPrefix": "aou-rw-stable-",
+    "registeredDomainName": "all-of-us-registered-stable",
+    "enforceRegistered": false
+  },
+  "cdr": {
+    "defaultCdrVersion": "Test Registered CDR"
+  },
+  "googleCloudStorageService": {
+    "credentialsBucketName": "aou-res-workbench-stable-credentials"
+  },
+  "googleDirectoryService": {
+    "gSuiteDomain": "stable.fake-research-aou.org"
+  },
+  "server": {
+    "stackdriverApiKey": "AIzaSyDPoX4Eg7-_FVKi7JFzEKaJpZ4IMRLaER4",
```

Comment:
Are we sure this isn't a secret?

Reply:
It's necessary to include in order to submit logs from the client -- but we should include a comment and ensure we're following best practices (e.g. https://support.google.com/cloud/answer/6310037)

Reply:
Totally makes sense.

Reply:
I'l also say from reading the docs that it doesn't appear possible to restrict the actual API calls that a given key can be used for. I need to add play with this to convince myself that it's restricted as expected.

Reply:
I'l also say from reading the docs that it doesn't appear possible to restrict the actual API calls that a given key can be used for. I need to add play with this to convince myself that it's restricted as expected.

Reply:
@calbach looking into this, I can confirm that using an API key I'm able to write log entries, but (as I intuitively expected) not able to do other operations (e.g. reading log entries). However from reviewing the docs at:

https://cloud.google.com/logging/docs/reference/v2/rest/v2/entries/write
_vs._
https://cloud.google.com/logging/docs/reference/v2/rest/v2/entries/list

I can't tell *why* this is the case. Neither one mentions that API keys do (or don't) work, and it's not obvious from the discussion of OAuth scopes on those pages either. How do I know what my API key is and isn't good for?

https://cloud.google.com/docs/authentication/api-keys purports to provide a list of only six APIs that API keys can be used for:

```
They are only supported by the following GCP services:

Google Cloud Natural Language API
Google Cloud Speech API
Google Cloud Translation API
Google Cloud Vision API
Google Cloud Endpoints
Google Cloud Billing Catalog API
```

... but that's obviously not complete, since these keys work for writing stackdriver entries, too.

Reply:
Ah, I see that https://cloud.google.com/error-reporting/reference/rest/v1beta1/projects.events/report explicitly calls out support for API keys.

Reply:
Ah, I see that https://cloud.google.com/error-reporting/reference/rest/v1beta1/projects.events/report explicitly calls out support for API keys.

Reply:
So, this might be a nice place to add a comment regarding the above. JSON doesn't allow comments. Worth moving these over to YAML?

Reply:
I'm "meh" on this, but wouldn't oppose it. Hacky alternatives include:

* longer more descriptive name like `publicApiKeyForErrorReports`
* `_comment` property next to the key

Reply:
I always prefer a rename for clarity over a comment. I was trying to add an issue to change this but can't right now.